frontend/src/components/search/searchbar/SearchBarView.jsx

```diff
@@ -0,0 +1,43 @@
+import TextField from '@mui/material/TextField'
+import InputAdornment from '@mui/material/InputAdornment'
+import SearchIcon from '@mui/icons-material/Search'
+
+const SearchBarView = ({ data }) => {
```

Comment:
Perhaps adding some sort of documentation on top regarding the props do, just as info for someone using this component later :)